apps/mac/FluentUIReactNative-macOS/Info.plist

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CFBundleDevelopmentRegion</key>
+	<string>$(DEVELOPMENT_LANGUAGE)</string>
+	<key>CFBundleExecutable</key>
+	<string>$(EXECUTABLE_NAME)</string>
+	<key>CFBundleIconFile</key>
+	<string></string>
+	<key>CFBundleIdentifier</key>
+	<string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
+	<key>CFBundleInfoDictionaryVersion</key>
+	<string>6.0</string>
+	<key>CFBundleName</key>
+	<string>$(PRODUCT_NAME)</string>
+	<key>CFBundlePackageType</key>
+	<string>$(PRODUCT_BUNDLE_PACKAGE_TYPE)</string>
+	<key>CFBundleShortVersionString</key>
+	<string>1.0</string>
+	<key>CFBundleVersion</key>
+	<string>1</string>
+	<key>LSMinimumSystemVersion</key>
+	<string>$(MACOSX_DEPLOYMENT_TARGET)</string>
+	<key>NSHumanReadableCopyright</key>
+	<string>Copyright © 2020 Microsoft. All rights reserved.</string>
```

Comment:
Per other comments, we should scrub this copyright and make sure we get the right legalese

Reply:
This is the same copyright we use in Polyester, but with the year updated. Why do you think it's wrong?

Reply:
This is public GitHub. Polyester is internal and not strictly licensed that I know of

Reply:
FWIW, fabric-ios just removed the human readable copyright from the info plist completely.

Reply:
I noticed fabric-macos has the copyright info still in this file

https://office.visualstudio.com/ISS/_git/fabric-macos?path=%2FOfficeUIFabricTestApp%2FOfficeUIFabricTestApp-Info.plist

Reply:
I'll go ahead and remove it though since fabric-ios has been in the public for longer, so more likely their legalese is correct.

Reply:
Yeah fabric-macos is not open source or public, so not a good example to follow.